p2p_message_format.proto

```diff
@@ -56,12 +56,18 @@ message Event {
     GetPKAckData get_pk_ack_data = 13;
     RePublishDIDAckData republish_ack_data = 14;
     ReconnectAckData reconnect_ack_data = 15;
+
+    ServiceRequestedAckData service_requested_ack_data = 16;
   }
 }
 
 message ChargingStatusData { float progress = 1; }
 
 message ServiceAckData { Response resp = 1; }
+message ServiceRequestedAckData {
+  Response resp = 1;
+  int64 wait_time = 2;
```

Comment:
I don't think this is needed. Sending incremental progress should be enough.
100% should means that it has been completed. There's currently no requirement for this.

Reply:
In the current implementation, one status thread controller handles two threads (the charging status thread and the charging stopper thread). Therefore, the first reason I need wait_time is to inform the charging_stopper when to stop the charging process. 
Here is [the detailed flow](https://peaq.slab.com/posts/v-2-charging-status-flow-2bmnl2iy). 

One more reason is if we send the wait_time back, users can know when the charging process stops automatically.

The charging status didn't change, and it still had the progress only.